google-cloud-tools-plugin/src/com/google/gct/idea/debugger/CloudDebugProcess.java

```diff
@@ -261,7 +261,7 @@ public void run() {
   /**
    * Finds the snapshot associated with the given id and sets it as the active snapshot in the current debug session.
    */
-  public void navigateToSnapshot(@NotNull final String id) {
+  public void navigateToSnapshot(@NotNull final String id, final boolean isSelectedBeforeTrigger) {
```

Comment:
could you add javadoc for isSelectBeforeTrigger?  Typically boolean flags to a method to change its behavior is discouraged for public API.